.changeset/forty-moose-pay.md

```diff
@@ -0,0 +1,9 @@
+---
+'@shopify/theme-check-common': minor
+'@shopify/theme-check-node': minor
+---
+
+Theme check verifies if setting key exists in schema
```

Comment:
Should we specify that this is for theme blocks? Or does this apply for other asset types as well?